pxtblocks/blocklyloader.ts

```diff
@@ -362,6 +362,22 @@ namespace pxt.blocks {
                 });
             }
             else {
+                // if requested, wrap block into a "set variable block"
+                if (fn.attributes.blockSetVariable && fn.retType) {
+
+                    const setblock = Blockly.Xml.textToDom(`
```

Comment:
Escape these strings

Reply:
Also, I don't think you need to specify the variable type field

Reply:
i use ``fn.retType.toLowerCase()`` later on for the variable name, just ensuring nothing crashes here.